src/components/Login/Login.css

```diff
@@ -44,13 +44,16 @@ button.submitButton {
   font-size: var(--font-size-medium);
 }
 
+.linksWrapper {
+  margin-top: 10px;
```

Comment:
This seems like a good spot for `1rem` or `var(--gutter)` instead of a hard-coded pixel value. Any reason for 10px specifically?

Reply:
Yeah, you are right. Thank you. I'll fix this in a moment.